api/src/client/DynamicsBusinessAddressClient.test.ts

```diff
@@ -0,0 +1,182 @@
+    jest.resetAllMocks();
+    logger = LogWriter("NavigatorWebService", "ApiLogs", "us-test-1");
+    ORIGINAL_ORG_URL = process.env.DCA_DYNAMICS_ORG_URL;
+    process.env.DCA_DYNAMICS_ORG_URL = ORG_URL;
+
+    client = DynamicsBusinessAddressClient(logger, process.env.DCA_DYNAMICS_ORG_URL);
+  });
+
+  afterEach(() => {
+    process.env.DCA_DYNAMICS_ORG_URL = ORIGINAL_ORG_URL;
+  });
+
+  const mockAccessToken = "access-granted";
+  const mockBusinessIds = ["id2", "id3"];
+
+  it("makes a get request to dynamics api with business address query and auth token in header", async () => {
+    const businessAddressMockResponse = {
+      value: [
+        {
+          createdo: "2023-05-31T10:31:51Z",
+          rgb_city: "Washington",
+          rgb_county: null,
+          rgb_name: "Apt GFG 3900A Watson Pl NW",
+          rgb_state: "DC",
+          rgb_street1: "3900A Watson Pl NW",
+          rgb_street2: "Apt GFG",
+          rgb_typecode: 100000000,
+          rgb_zip: "20016-6739",
+          statecode: 0,
+          rgb_addressid: "ff166755-9eff-ed11-a81b-001dd80703e0",
+        },
+        {
+          createdon: "2023-05-31T10:32:04Z",
+          rgb_city: "Chesapeake",
+          rgb_county: null,
+          rgb_name: "Trlr T 322 Ballahack Rd",
+          rgb_state: "VA",
+          rgb_street1: "Trlr T 322 Ballahack Rd",
+          rgb_street2: null,
+          rgb_typecode: 100000001,
+          rgb_zip: "23322-2479",
+          statecode: 0,
+          rgb_addressid: "4da7455c-9eff-ed11-a81b-001dd80703e0",
+        },
+      ],
+    };
+
+    mockAxios.get.mockResolvedValue({ data: { ...businessAddressMockResponse } });
+    expect(await client.getMappedBusinessAddressesByIds(mockAccessToken, mockBusinessIds)).toEqual([
+      {
+        id2: [
+          generateLicenseSearchAddress({
+            addressLine1: "3900A Watson Pl NW",
+            addressLine2: "Apt GFG",
+            zipCode: "20016",
+          }),
+          generateLicenseSearchAddress({
+            addressLine1: "Trlr T 322 Ballahack Rd",
+            addressLine2: "",
+            zipCode: "23322",
+          }),
+        ],
+      },
+      {
+        id3: [
+          generateLicenseSearchAddress({
+            addressLine1: "3900A Watson Pl NW",
+            addressLine2: "Apt GFG",
+            zipCode: "20016",
+          }),
+          generateLicenseSearchAddress({
+            addressLine1: "Trlr T 322 Ballahack Rd",
+            addressLine2: "",
+            zipCode: "23322",
+          }),
+        ],
+      },
+    ]);
+    expect(mockAxios.get).toHaveBeenCalledWith(
+      `${ORG_URL}/api/data/v9.2/rgb_addresses?$select=createdon,rgb_city,rgb_county,rgb_name,rgb_state,rgb_street1,rgb_street2,rgb_typecode,rgb_zip,statecode&$filter=(_rgb_businessid_value eq ${mockBusinessIds[0]})&$top=50`,
+      {
+        headers: {
+          Authorization: `Bearer ${mockAccessToken}`,
+        },
+      }
+    );
+    expect(mockAxios.get).toHaveBeenCalledWith(
```

Comment:
[dust] If it helps, there's a matcher called `toHaveBeenNthCalledWith` where you can assert which one was called first and second. I don't know if you care the order. But it also helps readability for me to be like, "oh yeah, both of these were supposed to have been called" instead of thinking "wait did we just do the same assertion twice" and then realizing